packages/core/src/integrations/requestdata.ts

```diff
@@ -1,61 +1,49 @@
 import { defineIntegration } from '../integration';
-import type { IntegrationFn } from '../types-hoist';
-import { type AddRequestDataToEventOptions, addNormalizedRequestDataToEvent } from '../utils-hoist/requestdata';
+import type { Event, IntegrationFn, RequestEventData } from '../types-hoist';
+import { parseCookie } from '../utils/cookie';
+import { getClientIPAddress, ipHeaderNames } from '../vendor/getIpAddress';
 
-export type RequestDataIntegrationOptions = {
+interface RequestDataIncludeOptions {
+  cookies?: boolean;
+  data?: boolean;
+  headers?: boolean;
+  ip?: boolean;
+  query_string?: boolean;
+  url?: boolean;
+}
+
+type RequestDataIntegrationOptions = {
   /**
-   * Controls what data is pulled from the request and added to the event
+   * Controls what data is pulled from the request and added to the event.
    */
-  include?: {
-    cookies?: boolean;
-    data?: boolean;
-    headers?: boolean;
-    ip?: boolean;
-    query_string?: boolean;
-    url?: boolean;
-  };
+  include?: RequestDataIncludeOptions;
 };
 
-const DEFAULT_OPTIONS = {
-  include: {
-    cookies: true,
-    data: true,
-    headers: true,
-    ip: false,
-    query_string: true,
-    url: true,
-  },
-  transactionNamingScheme: 'methodPath' as const,
-};
+const DEFAULT_INCLUDE = {
+  cookies: true,
+  data: true,
+  headers: true,
+  ip: false,
+  query_string: true,
+  url: true,
+} satisfies RequestDataIncludeOptions;
```

Comment:
I think in these cases it makes more sense to do the classical variable typing (`const DEFAULT_INCLUDE: RequestDataIncludeOptions = ...`). It will tell you when you have extra fields that are unnecessary to the type. We should only use satisfies when we want to specifically allow extra fields.